website/docs/installation/weave-gitops-enterprise/index.mdx

```diff
@@ -129,6 +138,7 @@ flux bootstrap github \
   --branch=main \
   --path=./clusters/management \
   --personal
+  --components source-controller,kustomize-controller,helm-controller,notification-controller,image-reflector-controller,image-automation-controller
```

Comment:
```suggestion
  --components-extra image-reflector-controller,image-automation-controller
```
Flux installs source-controller, kustomize-controller, helm-controller and notification-controller by default so I'd suggest using this flag to specify only the additional components.